test/dlc_tests/conftest.py

```diff
@@ -350,6 +350,18 @@ def pt14_and_above_only():
     pass
 
 
+@pytest.fixture(autouse=True)
+def log_current_test():
+    """
+    Log the name of the test currently being executed by pytest
+    """
+    try:
+        test_name = f"{os.environ.get('PYTEST_CURRENT_TEST').split('::')[-1].split(' ')[0]}"
+        LOGGER.info(f"============================= Executing test :: {test_name} :: =============================")
```

Comment:
Does each test set the common environment variable when it is begun even if the test is being executed in parallel with other tests in other threads?

Multiple processes would have separate env variables, but would the multiple parallel pytest threads be able to accurately print out their test name from a shared environment variable?

Reply:
It looks like each test gets its own process, and not its own thread -- https://docs.pytest.org/en/stable/example/simple.html#dynamically-adding-command-line-options

So I guess you can ignore this comment.